admin/category_product_listing.php

```diff
@@ -1001,42 +1001,44 @@
                     <?php
                   }
                   /* TODO: replace following form with anchors. */
+                  /* the parameter $search_parameter is PURPOSEFULLY left off the following 2 links.  */
+                  /* the function zen_get_all_get_params will pick up the $_GET search vars and include them in the string concatenation */
                   if ($product['products_status'] === '1') { ?>
-                    <a href="<?php echo zen_href_link(FILENAME_CATEGORY_PRODUCT_LISTING, 'action=setflag&flag=0&pID=' . $product['products_id'] . '&cPath=' . $cPath . (isset($_GET['page']) ? '&page=' . $_GET['page'] : '') . $search_parameter); ?>" role="button">
-                  <?php echo zen_icon('enabled', IMAGE_ICON_STATUS_ON, size: 'lg', hidden: true) ?>
+                    <a href="<?= zen_href_link(FILENAME_CATEGORY_PRODUCT_LISTING, zen_get_all_get_params(['action', 'pID', 'cPath',]) . 'action=setflag&flag=0&pID=' . $product['products_id'] . '&cPath=' . $cPath) ?>" role="button">
```

Comment:
Should `'flag'` be included in the function's exclusion list?

Reply:
nope.